lib/compress.js

```diff
@@ -2893,7 +2920,7 @@ merge(Compressor.prototype, {
                 });
             }
         }
-        return self;
+        return self.evaluate(compressor)[0];
```

Comment:
@alexlamsl Your grasp of the evaluate code is better than mine. What you write makes sense. I clearly wasn't testing what I thought I was testing.